cfn-resources/api-key/test/apikey.create-sample-cfn-request.sh

```diff
@@ -11,7 +25,7 @@ set -o pipefail
 profile="default"
 orgId="${MONGODB_ATLAS_ORG_ID}"
 
-projectName="cfn-bot-apikey-test"
```

Comment:
`cfn-bot-apikey-test` secret was already existing in AWS account and can't be deleted for 7 days (AWS policy on secret deletion), hence had to create a new one.